middlewares/validators/events.js

```diff
@@ -100,11 +100,29 @@ const endActiveEvent = async (req, res, next) => {
   }
 };
 
+const addPeerToEvent = async (req, res, next) => {
+  const schema = joi.object({
+    name: joi.string().required(),
```

Comment:
Are we having any text validations for these in the backend?

Reply:
what do you mean by text validation?
here joi is working as a validator.
Anything I am missing?